Looks like lint failed: 
```
app/containers/banner/helpusBanner.tsx[9, 39]: unused variable: 'KeyValue'

app/containers/helpus.tsx[5, 39]: unused variable: 'KeyValue'

Makefile:34: recipe for target 'lint' failed
```

LGTM though

--- a/ui/next/app/containers/banner/disconnectedBanner.tsx
+++ b/ui/next/app/containers/banner/disconnectedBanner.tsx
@@ -6,7 +6,7 @@ import { refreshHealth, HealthState } from "../../redux/health";
 
 class DisconnectedBannerProps {
   health: HealthState;
-  refreshHealth: () => void;
+  refreshHealth: typeof refreshHealth;
 }
 
 class DisconnectedBannerState {

--- a/ui/next/app/containers/banner/helpusBanner.tsx
+++ b/ui/next/app/containers/banner/helpusBanner.tsx
@@ -6,7 +6,7 @@ import { createSelector } from "reselect";
 import _ = require("lodash");
 
 import Banner from "./banner";
-import { KEY_HELPUS, OptInAttributes, KeyValue, loadUIData, saveUIData } from "../../redux/uiData";
+import { KEY_HELPUS, OptInAttributes, loadUIData, saveUIData } from "../../redux/uiData";
 import { setUISetting } from "../../redux/ui";
 
 export let HELPUS_BANNER_DISMISSED_KEY = "banner/helpusBanner/DISMISSED";
@@ -15,9 +15,9 @@ class HelpusBannerProps {
   attributes: OptInAttributes;
   attributesLoaded: boolean;
   dismissed: boolean = false;
-  loadUIData: (...keys: string[]) => void;
-  saveUIData: (...values: KeyValue[]) => void;
-  setUISetting: (key: string, value: any) => void;
+  loadUIData: typeof loadUIData;
+  saveUIData: typeof saveUIData;
+  setUISetting: typeof setUISetting;
 }
 
 class HelpusBanner extends React.Component<HelpusBannerProps, {}> {

--- a/ui/next/app/containers/clusterOverview.tsx
+++ b/ui/next/app/containers/clusterOverview.tsx
@@ -20,7 +20,7 @@ interface ClusterMainProps {
     availableCapacity: number;
     bytesUsed: number;
   };
-  refreshNodes(): void;
+  refreshNodes: typeof refreshNodes;
 }
 
 /**

--- a/ui/next/app/containers/databases/databaseDetails.tsx
+++ b/ui/next/app/containers/databases/databaseDetails.tsx
@@ -143,9 +143,9 @@ interface DatabaseMainData {
  */
 interface DatabaseMainActions {
   // Call if the user indicates they wish to change the sort of the table data.
-  setUISetting(key: string, value: any): void;
+  setUISetting: typeof setUISetting;
   // Call when the data needs to be refreshed.
-  refreshDatabaseDetails(id: string): void;
+  refreshDatabaseDetails: typeof refreshDatabaseDetails;
 }
 
 /**

--- a/ui/next/app/containers/databases/databaseList.tsx
+++ b/ui/next/app/containers/databases/databaseList.tsx
@@ -80,9 +80,9 @@ interface DatabasesMainData {
  */
 interface DatabasesMainActions {
   // Call if the user indicates they wish to change the sort of the table data.
-  setUISetting(key: string, value: any): void;
+  setUISetting: typeof setUISetting;
 
-  refreshDatabases(): void;
+  refreshDatabases: typeof refreshDatabases;
 }
 
 /**

--- a/ui/next/app/containers/databases/tableDetails.tsx
+++ b/ui/next/app/containers/databases/tableDetails.tsx
@@ -251,9 +251,9 @@ interface TableMainData {
  */
 interface TableMainActions {
   // Call if the user indicates they wish to change the sort of any table data.
-  setUISetting(key: string, value: any): void;
+  setUISetting: typeof setUISetting;
   // Refresh the table data
-  refreshTableDetails(database: string, table: string): void;
+  refreshTableDetails: typeof refreshTableDetails;
 }
 
 // params from the route for the table name and database name

--- a/ui/next/app/containers/events.spec.tsx
+++ b/ui/next/app/containers/events.spec.tsx
@@ -7,11 +7,12 @@ import * as sinon from "sinon";
 
 import * as protos from  "../js/protos";
 import { EventList, EventRow } from "./events";
+import { refreshEvents } from "../redux/events";
 
 type Event = cockroach.server.serverpb.EventsResponse.Event;
 
-function makeEventList(events: Event[], refreshEvents: () => void) {
-  return shallow(<EventList events={events} refreshEvents={refreshEvents}></EventList>);
+function makeEventList(events: Event[], refreshEventsFn: typeof refreshEvents) {
+  return shallow(<EventList events={events} refreshEvents={refreshEventsFn}></EventList>);
 }
 
 function makeEvent(event: Event) {

--- a/ui/next/app/containers/events.tsx
+++ b/ui/next/app/containers/events.tsx
@@ -64,7 +64,7 @@ export class EventRow extends React.Component<EventRowProps, {}> {
 
 export interface EventListProps {
   events: Event[];
-  refreshEvents: () => void;
+  refreshEvents: typeof refreshEvents;
 };
 
 export class EventList extends React.Component<EventListProps, {}> {

--- a/ui/next/app/containers/helpus.tsx
+++ b/ui/next/app/containers/helpus.tsx
@@ -2,15 +2,15 @@
 import * as React from "react";
 import _ = require("lodash");
 import { connect } from "react-redux";
-import { KEY_HELPUS, OptInAttributes, KeyValue, loadUIData, saveUIData } from "../redux/uiData";
+import { KEY_HELPUS, OptInAttributes, loadUIData, saveUIData } from "../redux/uiData";
 import { setUISetting } from "../redux/ui";
 import { HELPUS_BANNER_DISMISSED_KEY } from "./banner/helpusBanner";
 
 export interface HelpUsProps {
   optInAttributes: OptInAttributes;
-  loadUIData: (...keys: string[]) => void;
-  saveUIData: (...values: KeyValue[]) => void;
-  setUISetting: (key: string, value: any) => void;
+  loadUIData: typeof loadUIData;
+  saveUIData: typeof saveUIData;
+  setUISetting: typeof setUISetting;
 }
 
 /**

--- a/ui/next/app/containers/nodeOverview.tsx
+++ b/ui/next/app/containers/nodeOverview.tsx
@@ -11,7 +11,7 @@ import { Bytes, Percentage } from "../util/format";
 
 interface NodeOverviewProps extends RouteComponentProps<any, any> {
   node: NodeStatus;
-  refreshNodes(): void;
+  refreshNodes: typeof refreshNodes;
 }
 
 /**

--- a/ui/next/app/containers/nodesOverview.tsx
+++ b/ui/next/app/containers/nodesOverview.tsx
@@ -190,9 +190,9 @@ interface NodesMainData {
  */
 interface NodesMainActions {
   // Call if the nodes statuses are stale and need to be refreshed.
-  refreshNodes(): void;
+  refreshNodes: typeof refreshNodes;
   // Call if the user indicates they wish to change the sort of the table data.
-  setUISetting(key: string, value: any): void;
+  setUISetting: typeof setUISetting;
 }
 
 /**

--- a/ui/next/app/containers/raftRanges.tsx
+++ b/ui/next/app/containers/raftRanges.tsx
@@ -29,7 +29,7 @@ interface RangesMainData {
  */
 interface RangesMainActions {
   // Call if the ranges statuses are stale and need to be refreshed.
-  refreshRaft(): void;
+  refreshRaft: typeof refreshRaft;
 }
 
 interface RangesMainState {

--- a/ui/next/app/containers/timescale.tsx
+++ b/ui/next/app/containers/timescale.tsx
@@ -10,7 +10,7 @@ import * as timewindow from "../redux/timewindow";
 interface TimeScaleSelectorProps {
   currentScale: timewindow.TimeScale;
   availableScales: timewindow.TimeScaleCollection;
-  setTimeScale: (ts: timewindow.TimeScale) => void;
+  setTimeScale: typeof timewindow.setTimeScale;
 }
 
 interface TimeScaleSelectorState {

--- a/ui/next/app/containers/timewindow.tsx
+++ b/ui/next/app/containers/timewindow.tsx
@@ -10,7 +10,7 @@ interface TimeWindowManagerProps {
   // The current timewindow redux state.
   timeWindow: timewindow.TimeWindowState;
   // Callback function used to set a new time window.
-  setTimeWindow: (tw: timewindow.TimeWindow) => void;
+  setTimeWindow: typeof timewindow.setTimeWindow;
   // Optional override method to obtain the current time. Used for tests.
   now?: () => moment.Moment;
 }

